knative-operator/pkg/controller/knativeserving/kourier/kourier.go

```diff
@@ -0,0 +1,37 @@
+package kourier
+
+import (
+	mf "github.com/jcrossley3/manifestival"
+	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/common"
+	servingv1alpha1 "knative.dev/serving-operator/pkg/apis/serving/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var log = common.Log.WithName("kourier")
+
+func ApplyKourier(instance *servingv1alpha1.KnativeServing, api client.Client) error {
+	log.Info("Installing Kourier Ingress")
+	resources := []string{"deploy/resources/kourier/kourier-latest.yaml", "deploy/resources/kourier/kourier-openshift.yaml"}
+	for _, path := range resources {
+		if err := apply(instance, api, path); err != nil {
+			log.Error(err, "Failed to apply %s: %v", path, err)
+			return err
+		}
+	}
+	return nil
+}
+
+func apply(instance *servingv1alpha1.KnativeServing, api client.Client, path string) error {
+	manifest, err := mf.NewManifest(path, false, api)
+	if err != nil {
+		return err
+	}
+	transforms := []mf.Transformer{mf.InjectOwner(instance)}
```

Comment:
Does this give us a cross-namespace ownerRef? Do we need to use a finalizer instead?

Reply:
Ah, yes... This is a owner for knative-serving in `knative-serving` ns. So we need to use a finalizer instead.